src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -853,7 +853,7 @@ def clone_subvolume_snapshot(self, **kwargs):
 
     def _get_src_path(self, vol_handle, dst_group, dst_subvol):
         src_subvol_details = dst_subvol._get_clone_source()
-        src_group_name = src_subvol_details.get('group', dst_group.NO_GROUP_NAME)
+        src_group_name = src_subvol_details.get('group', Group.NO_GROUP_NAME)
```

Comment:
nit: src_subvol_details.get('group', None)

That way `Group` need not be imported 👍

Reply:
Done.